### ISSUE TITLE
Supporting Apartment-Style Living

### ISSUE THREAD

**Issue by LCBradley3k** [google/open-location-code#148]
Food for thought, but I see a problem when it comes to apartment-style living, especially one with a lot of floors. It might be of interest to look into solutions when say a building has 6 floors and 12 different families live in it. With Open Location Code, they would all fall into the same precise square - yet need to somehow be made unique.

**Comment by bocops** [comment]
@LCBradley3k, things like these are regularly brought up - there's even a statement in the FAQ about it:

_Q: Why don't Open Location Codes include altitude?
A: We didn't want to append it as a suffix or bury it in the code because we
want to be able to truncate the codes reliably. We also didn't want to
unnecessarily extend the length of codes for what we expect to be a minority
of cases. Altitude is more useful to people as a floor number, but there are
different ways of numbering building floors depending on local custom._

_In summary, we couldn't think of a way that was better than specifying the
code and allowing people to just say "3rd floor"._

Open Location Code is great to get you to the front door of a building. Once there, there are other means going forward. For example, if there are twelve families sharing the same entry (of an apartment complex), chances are there's a door bell nameplate with all the different names on it. :)

**Comment by sunk818** [comment]
Related discussion: https://github.com/google/open-location-code/issues/25
